spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarProperties.java

```diff
@@ -1609,9 +1610,9 @@ public void setTlsProtocols(Set<String> tlsProtocols) {
 		}
 
 		public Map<String, Object> buildProperties() {
-			if (!StringUtils.hasText(this.getAuthParams()) && !CollectionUtils.isEmpty(this.getAuthentication())) {
+			if (StringUtils.hasText(this.getAuthParams()) && !CollectionUtils.isEmpty(this.getAuthentication())) {
```

Comment:
This was another bug that was unrelated to the auth props not being respected, but ultimately this prevented any auth props from being set at all.